openslides/core/static/js/core/projector.js

```diff
@@ -56,6 +56,30 @@ angular.module('OpenSlidesApp.core.projector', ['OpenSlidesApp.core'])
     }
 ])
 
+.controller('LanguageCtrl', [
+    '$scope',
+    'gettextCatalog',
+    'Languages',
+    'Config',
+    'ProjectorID',
+    function ($scope, gettextCatalog, Languages, Config, ProjectorID) {
+        // for the dynamic title
+        $scope.projectorId = ProjectorID();
```

Comment:
I think the ProjectorContainerCtrl is a child of the LanguageCtrl because it is attached inside the html tag. So the scope of the ProjectorContainerCtrl inherits the scope of the LanguageCtrl. So our scope var has not to be applied again.